integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/CatalogsPageTest.java

```diff
@@ -47,6 +47,7 @@ public class CatalogsPageTest extends AbstractWebIT {
   protected static String hdfsUri = "hdfs://127.0.0.1:9000";
   protected static String mysqlUri = "jdbc:mysql://127.0.0.1";
   protected static String postgresqlUri = "jdbc:postgresql://127.0.0.1";
+  protected static String kafkaUri = "http://127.0.0.1:9092";
```

Comment:
Please don't forget to replace the container's actual address when checking Kafka topics in subsequent verification. Currently, there are no restrictions on creating a catalog.